blob/blob_test.go

```diff
@@ -42,7 +42,7 @@ func TestBlob(t *testing.T) {
 		{
 			name: "verify nID",
 			expectedRes: func(t *testing.T) {
-				require.NoError(t, apptypes.ValidateBlobNamespaceID(blob[0].NamespaceID()))
+				require.NoError(t, apptypes.ValidateBlobNamespaceID(blob[0].Namespace()))
```

Comment:
[no change needed] oof `ValidateBlobNamespaceID` should be renamed so created https://github.com/celestiaorg/celestia-app/issues/1849